app/views/admin_mailer/send_spam_alert.text.erb

```diff
@@ -0,0 +1,15 @@
+<% content_for :message do %>
+
+The following accounts have a suspicious level of traffic:<
+ <%@spam_score.sort_by{|k,v| -1*v}.map do |pseud,score| %>
```

Comment:
Spaces! The final frontier!

(Although it's a good idea to skip indenting in plain text emails, so you should remove the leading space on this line and on lines 5, 6, 8, 9, and 11. They can stick around on line 10, though.)